server/api/mock.ts

```diff
@@ -1,10 +1,35 @@
 import url from "node:url";
 
 export default defineEventHandler(async (event) => {
+    const method = event.node.req.method;
+    const code = event.node.res.statusCode;
     const requestUrl = new url.URL("http://localhost" + event.node.req.url);
-    return {
+    
+    let responseData = {
         title: "Success",
-        message: "Mock request processed successfully",
+        method: method,
         params: requestUrl.searchParams,
```

Comment:
`searchParams` is empty on GET request, no matter what is sent.
Check this out to maybe tweak it https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams.